website/source/docs/providers/aws/r/api_gateway_method.html.markdown

```diff
@@ -39,7 +39,8 @@ The following arguments are supported:
 * `rest_api_id` - (Required) The ID of the associated REST API
 * `resource_id` - (Required) The API resource ID
 * `http_method` - (Required) The HTTP Method (`GET`, `POST`, `PUT`, `DELETE`, `HEAD`, `OPTION`)
-* `authorization` - (Required) The type of authorization used for the method
+* `authorization` - (Required) The type of authorization used for the method (`NONE`, `CUSTOM`)
```

Comment:
AWS_IAM is also a valid authorisation type (authorizer_id not required for this)